src/axom/slic/docs/sphinx/sections/appendix.rst

```diff
@@ -52,6 +52,66 @@ logging within an application.
   functionality in application specific macros to better suit the requirements
   of the application.
 
+.. _CollectiveSlicMacros:
+
+Collective Slic Macros
+^^^^^^^^^^^^^^^^^^^^^^^^^
+A subset of SLIC macros are collective operations when used with
+MPI-aware :ref:`logStream` instances such as :ref:`SynchronizedStream`
+or :ref:`LumberjackStream`.
+
+Additionally, macros such as ``SLIC_WARNING`` and ``SLIC_CHECK`` become collective
+operations when certain flags are toggled on or functions are called. Other macros
+such as ``SLIC_ERROR`` and ``SLIC_ASSERT`` can be made not collective when certain
+functions are called.
+
+The table below details which SLIC macros are collective:
+
+.. list-table:: Title
```

Comment:
```suggestion
.. list-table:: Macros, Collective and Non-Collective
```

Additionally, throughout the table I'd change the word "when" to "after" (or perhaps "if," but I like that one less), although that's more of a style change to add specificity.